synfig-studio/src/synfigapp/main.cpp

```diff
@@ -109,7 +109,7 @@ synfigapp::Main::Main(const synfig::String &basepath, synfig::ProgressCallback *
 
 #ifdef ENABLE_NLS
 	String locale_dir;
-	locale_dir = etl::dirname(basepath)+ETL_DIRECTORY_SEPARATOR+"share"+ETL_DIRECTORY_SEPARATOR+"locale";
+	locale_dir = basepath+ETL_DIRECTORY_SEPARATOR+"share"+ETL_DIRECTORY_SEPARATOR+"locale";
```

Comment:
Please note `basepath` is passed to `synfig::Main` constructor too:
```
synfigapp::Main::Main(const synfig::String &basepath, synfig::ProgressCallback *cb):
	synfig::Main(basepath,cb),
```

Reply:
Ouch!